### DISCUSSION TITLE
Using FlxTween.color makes the image not visible

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2603] [Q&A]
This is how menu looks without using FlxTween.color
![image](https://user-images.githubusercontent.com/60806448/179388363-e8ee9d01-fe86-416c-9a05-66921209adcb.png)

Sky is the object that should be tweened (to make it look like time passes and sky color changes), skyTween is a function that provides it
![image](https://user-images.githubusercontent.com/60806448/179388411-8ac6ed70-fcbf-44e7-b4fb-31122a28ee8a.png)

This is nextPhase function, that's called when a tween finishes
![image](https://user-images.githubusercontent.com/60806448/179388423-a8841064-2689-406d-916e-9abbfc432454.png)

but if i un-comment skyTween() line in create(), then the sky will disappear completely

am i douing smth wrong?

**Comment by RichardBray** [comment]
Can you create a video of what it looks like when the tween happens? 
If colour tween is causing an issue you could always have two sprites, have one at an alpha of 0 and just tween between alphas, if that makes sense.
